components/Dialogs/src/Theming/MDCAlertController+MaterialTheming.m

```diff
@@ -23,6 +23,10 @@
 @implementation MDCAlertController (MaterialTheming)
 
 - (void)applyThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme {
+  // TODO(https://github.com/material-components/material-components-ios/issues/6637 ): Force-load
+  // the view before setting the buttons fonts to ensure that they aren't overwritten in
+  // -updateButtonFont. If the bug gets fixed, this hack can be removed.
+  (void)self.view;
```

Comment:
This feels like a hack. I would prefer a solution that is applied to the component (not the themers) so it works also when clients write their own custom theming extensions.